.cargo/config_fast_builds.toml

```diff
@@ -144,3 +144,8 @@ rustflags = [
 # In most cases the gains are negligible, but if you are on macos and have slow compile times you should see significant gains.
 # [profile.dev]
 # debug = 1
+
+# This is enables you to run the CI tool using `cargo ci`.
+# This is not enabled by default, you need to copy this file to `config.toml`.
+[alias]
+ci = "run --package ci --"
```

Comment:
You're missing a new line.

Reply:
I don't think so, it shows up in my editor.

<img width="711" alt="image" src="https://github.com/bevyengine/bevy/assets/59022059/ec99dca6-85b9-403f-9259-1b4b467884f8">

I think Github hides newlines at the end of the file.